tests/providers/odbc/hooks/test_odbc.py

```diff
@@ -17,35 +17,41 @@
 # specific language governing permissions and limitations
 # under the License.
 #
-import json
+import unittest
 from unittest import mock
 from urllib.parse import quote_plus, urlparse
 
 import pyodbc
+from parameterized import parameterized
 
 from airflow.models import Connection
 from airflow.providers.odbc.hooks.odbc import OdbcHook
 
 
-class TestOdbcHook:
-    def get_hook(self=None, hook_params=None, conn_params=None):
-        hook_params = hook_params or {}
-        conn_params = conn_params or {}
-        connection = Connection(
-            **{
-                **dict(login='login', password='password', host='host', schema='schema', port=1234),
-                **conn_params,
-            }
-        )
+class TestOdbcHook(unittest.TestCase):
+
+    def setUp(self):
+        super().setUp()
+
+        self.conn = conn = mock.MagicMock()
+        self.conn.login = 'login'
+        self.conn.password = 'password'
+        self.conn.host = 'host'
+        self.conn.schema = 'schema'
+        self.conn.port = 1234
```

Comment:
Please use the `Connection` class to init a connection like in the other cases.

For example:
```python
self.conn = mock.Mock(return_value=Connection(...))
```